app/src/components/market/sections/market_verify/option/kleros_curation.tsx

```diff
@@ -131,6 +164,10 @@ export const KlerosCuration: FC<Props> = (props: Props) => {
   let KlerosNotice
   let KlerosRightColumn
 
+  const urlSwitch = (url: string) => {
+    const win = window.open(url, '_blank')
+    win && win.focus()
+  }
```

Comment:
The problem with this approach is that [browser support is unknown](https://caniuse.com/?search=window.open). I think we're better off staying on the safe side and using what we know works for sure